pkg/networkmanager/firewall.jsx

```diff
@@ -43,7 +43,6 @@ import { FirewallSwitch } from "./firewall-switch.jsx";
 
 import { superuser } from "superuser";
 
-import "page.scss";
 import "form-layout.scss";
 import "./networking.scss";
```

Comment:
I'd like to understand this. So the page.scss import ought to move to pkg/networkmanager/networking.scss (but that doesn't currently import/use it) -- but why keep form-layout.scss here? (I know you eliminate it in another PR, but conceptually..)

And networking.scss is the top-level css which we should keep here?

Reply:
So this comes again to the point, that each page should just include one scss file and all the rest of the CSS/SCSS includes should happen in the scss code itself.  netwofking.scss is shared between network and firewall pages. `@use page` new line was introduced  in this PR  to the networking.scss, so now it includes it.

form-layout should be also moved to scss, but it will be removed anyway from the other PR indeed.